app/components/common/measureCommitteeList/measureCommitteeList.html

```diff
@@ -1,6 +1,6 @@
 <ul>
   <li ng-repeat="committee in committees">
-    <h4><a ui-sref="appMain.committee.main({committee_id: committee.committee_id})">{{ committee.name }}</a></h4>
+    <h4><a ui-sref="appMain.committee.main({filer_id: committee.filer_id})">{{ committee.name }}</a></h4>
```

Comment:
This reminds me, from the referendum money page, we're using `ng-href` instead of `ui-sref`, but luckily the url matches what the state change expects.